open-craft/xblock-vectordraw
# SE-1917 Fix flaky tests and add license

This adds a license and fixes flaky tests.

**Test instructions**:

- ensure tests consistently pass on python3 and any other versions we're still supporting.
- sanity check changes
- check license added is correct

**Reviewer**:
